app-backend/db/src/main/scala/filters/Filterables.scala

```diff
@@ -449,9 +449,26 @@ trait Filterables extends RFMeta with LazyLogging {
             params.projectFilterParams.projectType.map({ projectType =>
               fr"annotation_projects.project_type = $projectType"
             }),
+            params.capturedAt.map({ capturedAt =>
+              fr"annotation_projects.captured_at = $capturedAt"
```

Comment:
I'm curious about this -- do we want to require exact matches for a time field with this much precision? What's the interaction this parameter is supposed to facilitate?